draft-law-moq-cat4moqt.md

```diff
@@ -106,56 +106,72 @@ The "moqt" claim is defined by the following CDDL:
 
 ~~~~~~~~~~~~~~~
 $$Claims-Set-Claims //= (moqt-label => moqt-value)
-moqt-label = XXX TODO - how do we register this?
-moqt-value = [ + moqt-object ]
+moqt-label = TBD_MOQT
+moqt-value = [ + moqt-limit ]
+moqt-limit = [ moqt-actions, moqt-ns-match, moqt-track-match ]
+moqt-actions = moqt-action | [ + moqt-action ]
```

Comment:
wonder if we can make moq actions just an array with the base case being array of single element